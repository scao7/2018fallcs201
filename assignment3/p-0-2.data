8 8 180 ; 0 5 437 ; 9 6 170 ; 1 2 362 ; 8 1 137 ; 
2 4 167 ; 2 6 431 ; 7 8 219 ; 8 9 307 ; 1 7 146 ; 
2 9 444 ; 7 6 372 ; 5 9 285 ; 9 3 476 ; 1 2 319 ; 
7 9 250 ; 5 2 162 ; 3 4 427 ; 4 1 176 ; 3 9 304 ; 
2 3 425 ; 0 7 457 ; 1 2 86 ; 6 1 451 ; 8 0 191 ; 
9 0 64 ; 2 3 111 ; 8 0 308 ; 1 6 481 ; 0 2 17 ; 
5 7 112 ; 2 7 446 ; 7 3 456 ; 8 7 370 ; 8 0 196 ; 
9 4 277 ; 4 7 426 ; 6 5 280 ; 4 8 58 ; 6 7 419 ; 
5 9 361 ; 2 5 476 ; 8 3 341 ; 5 5 388 ; 5 2 298 ; 
1 9 38 ; 8 5 7 ; 5 6 423 ; 3 6 361 ; 2 8 145 ; 
1 9 283 ; 8 8 349 ; 1 9 3 ; 7 9 481 ; 1 7 86 ; 
2 5 213 ; 9 5 74 ; 7 0 132 ; 4 2 399 ; 2 7 4 ; 
3 3 316 ; 3 2 299 ; 2 5 47 ; 3 9 221 ; 2 7 190 ; 
0 1 151 ; 7 5 85 ; 7 8 214 ; 9 9 172 ; 9 4 368 ; 
2 0 222 ; 6 0 447 ; 3 9 21 ; 2 2 135 ; 0 2 179 ; 
1 5 153 ; 8 0 393 ; 8 5 110 ; 8 9 228 ; 2 9 459 ; 
4 1 98 ; 6 8 132 ; 6 1 410 ; 8 7 84 ; 3 6 338 ; 
8 1 260 ; 9 5 211 ; 6 5 470 ; 7 5 168 ; 6 5 58 ; 
9 5 440 ; 9 8 335 ; 1 0 393 ; 8 4 459 ; 6 5 310 ; 
7 5 283 ; 4 9 233 ; 0 1 42 ; 5 1 480 ; 2 0 367 ; 

