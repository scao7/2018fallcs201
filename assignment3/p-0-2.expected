0: 8
1: 1(8)137 3(8)341 4(8)58 5(8)7 6(8)132
2: 7(5)112 9(1)38
3: 0(9)64
4: 2(0)17
weight: 906
